README.md

```diff
@@ -59,31 +73,57 @@ Each of the directories only include the contents of the zip file that is sent t
 sure to sanitize `aws.env` and `secrets.env` files of all sensitive data, and update their extension to be
 `.env.example`.
 
-### `/data/apis/{oisTitle}/templates`
+### `/data/{protocol/{oisTitle}/templates`
 
 Hosts files containing template data and metadata. The files are named to describe what the template is for so that they
-are human-browseable. The files contents are:
+are human-browsable. The files contents are:
 
 - `airnode`: Airnode address (must match the one in `apiMetadata.json`)
 - `endpointId`: Endpoint ID (must match one from `config.json`)
 - `parameters`: Airnode ABI-encoded parameters
 - `templateId`: Template ID (must match `airnode`, `endpointId`, `parameters` encoded and hashed)
 - `decodedParameters`: `parameters` decoded to be human-readable
-- `chains`: Chains that the template is currently deployed on (must match ones from `chains.json`)
+- `versionedChains`: Chains that the template is currently deployed on (must match ones from `chains.json`), with
+  deployment versioning.
+
+---
+
+### `/data/proto-psp/{oisTitle}/beacons`
+
+Files containing subscription data and metadata. The files are named to describe what the subscription is for so that
+they are human-browsable.
+
+- `beaconId`: A subscription ID, eg. "0x168194af62ab1b621eff3be1df9646f198dcef36f9eace0474fd19d47b2e0039"
```

Comment:
Beacon ID or subscription ID?

Reply:
It should be a `beaconId`, unless you disagree? Airkeeper calls these `subscriptionIds`.

Reply:
We are using `beaconId` right? Tell me if I'm wrong/you disagree.